app/models/bank_holiday.rb

```diff
@@ -0,0 +1,18 @@
+class BankHoliday < ApplicationRecord
+  serialize :dates, Array
+
+  scope :by_updated_at, -> { order(updated_at: :asc) }
+  before_validation :populate_dates
+
+  validates :dates, presence: true
+
+  def self.dates
+    BankHolidayUpdateWorker.perform_in 10.seconds
+    instance = by_updated_at.last || create
```

Comment:
`create!` with an exclamation mark?
😃

Reply:
Damn! I was so pleased with myself for using `save!` in the worker. I even did a quick "Check for non-bang save methods before Julien's spots them" check.